.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,124 @@
+name: 🐛 Bug report
+description: Create a report to help us improve.
+labels:
+  - Bug
+
+body:
+  - type: textarea
+    attributes:
+      label: Bug Description
+      description: Please describe clearly and concisely what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected Behaviour
+      description: Please describe clearly and concisely what the expected behaviour should be.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Screenshots
+      description: >
+        Please provide detailed steps on how to reproduce the bug. Provide a URL where the issue can be seen on the
+        frontend when possible, otherwise go to “View source” in the browser and copy all to paste in a
+        [Gist](https://gist.github.com/) and share it.
+
+  - type: input
+    attributes:
+      label: PHP Version
+      description: Specify which PHP version the bug occurs on.
+      placeholder: e.g. 8.1
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Plugin Version
+      description: Specify which plugin version the bug occurs on.
+      placeholder: e.g. 2.1.4
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: AMP plugin template mode
+      description: Specify which template mode the bug occurs on.
+      options:
+        - Standard
+        - Transitional
+        - Reader
+    validations:
+      required: true
```

Comment:
The issue could occur on all template modes. Maybe this should be `checkboxes` instead? Or there could be an all option? Maybe it shouldn't be required.

Reply:
The [`dropdown`](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#dropdown) input has a `multiple` attribute that allows for the user to select multiple options, which I think would work.

Reply:
Done in 0901533f9e73437d8b7b571ed73c85dd023f1e36.